sql/mysqld.cc

```diff
@@ -6625,7 +6623,7 @@ struct my_option my_long_options[]=
    "mysql.gtid_slave_pos",
    &gtid_pos_auto_engines, 0, 0, GET_STR, REQUIRED_ARG,
    0, 0, 0, 0, 0, 0 },
-#ifdef HAVE_LARGE_PAGE_OPTION
+#ifdef HAVE_SOLARIS_LARGE_PAGES
```

Comment:
Unsure about this change. We normally don't remove options (or tighten support) without deprecating them first.